plugins/dev-tools/src/generateFieldTestBlocks.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A field test helper that generates blocks with the field in
+ * various configurations.
+ * @author samelh@google.com (Sam El-Husseini)
+ */
+
+import * as Blockly from 'blockly/core';
+
+
+/**
+ * Generates a number of field testing blocks for a specific field and returns
+ * the toolbox xml string.
+ * @param {string} fieldName The name of the field.
+ * @param {Object|Array.<Object>=} options An options object containing a label
+ *     and an args map that is passed to the field during initialization.  If an
+ *     array is passed, multiple groups of blocks are created each with
+ *     different initialization arguments.
+ * @return {string} The toolbox XML string.
+ */
+export default function generateFieldTestBlocks(fieldName, options) {
```

Comment:
I thought we wanted to avoid export default according to style guide.

Reply:
Good point.

Reply:
Done.

Reply:
+1 to avoiding export default.